padmin/non_persistent_topics_impl.go

```diff
@@ -105,6 +106,42 @@ type NonPersistentTopicsImpl struct {
 	options
 }
 
+func (n *NonPersistentTopicsImpl) GetStatsForTopic(tenant string, namespace string, topic string) (*TopicStatistics, error) {
```

Comment:
It appears that there may be some redundancy in the use of 'get' and 'for' in the 'stats' section of the code. Perhaps we should consider adopting some naming conventions from Java to improve readability and maintainability.